Preparation/MyStack/MyStackInter.cs

```diff
@@ -1,34 +1,27 @@
 using System;
 using System.Collections;
+using System.Data;
 
 namespace MyStack
 {
-    public class MyStack
+    public class MyStack : IEnumerable
     {
-        public static readonly int maxsize = 10;
+        public static int count = 1;
```

Comment:
Should not be static, each instance of collection should refer to it's own count